src/content/docs/nrql/get-started/introduction-nrql-new-relics-query-language.mdx

```diff
@@ -141,12 +141,12 @@ We use NRQL behind the scenes to generate many of the charts and dashboards in o
 />
 
 <figcaption>
-  Many of our charts and visualizations are built using NRQL. One way to get started using NRQL is to view a chart's query and then edit it to make your own custom chart.
+  We build many of our charts and visualizations using NRQL. You can view a chart's query and then edit it to make your own custom chart as a quick way to get started using NRQL.
```

Comment:
Is it worth clarifying that these are internal charts and visualizations? I think what you're trying to say here is: NRQL's so great, we use it all the time ourselves!

Reply:
Good call, modified the wording to make it clear that this is what we do ourselves.